projects/storefrontlib/src/cms-components/myaccount/my-interests/my-interests.component.html

```diff
@@ -0,0 +1,226 @@
+                            </div>
+                          </div>
+                        </ng-container>
+                        <div
+                          class="cx-property"
+                          *ngIf="
+                            interest.product.stock.stockLevelStatus ===
+                            'outOfStock'
+                          "
+                        >
+                          <div
+                            class="cx-label cx-product-interests-product-stock"
+                          >
+                            {{ 'myInterests.outOfStock' | cxTranslate }}
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </td>
+                <td>
+                  <div class="cx-product-interests-product-price">
+                    <div class="d-md-none cx-product-interests-label">
+                      {{ 'myInterests.price' | cxTranslate }}
+                    </div>
+                    <span>{{ interest.product.price.formattedValue }}</span>
+                  </div>
+                </td>
+                <td>
+                  <div class="cx-product-interests-subscriptions">
+                    <div class="d-md-none cx-product-interests-label">
+                      {{ 'myInterests.notifications' | cxTranslate }}
+                    </div>
+                    <div
+                      class="cx-product-interests-notification"
+                      *ngFor="
+                        let interestEntry of interest.productInterestEntry
+                      "
+                    >
+                      <span class="cx-product-interests-type">
+                        {{
+                          'myInterests.' + interestEntry.interestType
+                            | cxTranslate
+                        }}
+                      </span>
+                      <span class="cx-product-interests-expiration-date">
+                        {{
+                          'myInterests.expirationDate'
+                            | cxTranslate
+                              : {
+                                  expirationDate:
+                                    interestEntry.expirationDate | date
+                                }
+                        }}
+                      </span>
+                    </div>
+                  </div>
+                </td>
+                <td>
+                  <div class="cx-actions cx-product-interests-remove-button">
+                    <button
+                      type="button"
+                      class="link cx-product-interests-remove-btn"
+                      [class.disabled]="
+                        (getInterestsloading$ | async) ||
+                        (getRemoveInterestsloading$ | async)
+                      "
+                      [disabled]="
+                        (getInterestsloading$ | async) ||
```

Comment:
Do we need `getInterestsloading$` here, as it seems it's taken into account above?

Reply:
I think it is necessary, it is a deletion operation after all.